extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/MultiValueScrutinizerTest.java

```diff
@@ -19,6 +24,9 @@ public void testNoIssue() {
         ItemIdValue idB = TestingData.matchedId;
         ItemUpdate update = new ItemUpdateBuilder(idA).addStatement(TestingData.generateStatement(idA, idB))
                 .addStatement(TestingData.generateStatement(idA, idB)).build();
+        ConstraintFetcher fetcher = mock(ConstraintFetcher.class);
+        when(fetcher.hasMultiValue(anyObject())).thenReturn(true);
```

Comment:
I think the end goal is to remove any methods that are specific to a given constraint from the `ConstraintFetcher` interface. So I would rather mock a generic method (like `getSingleConstraint`) and make sure the scrunizer calls that.